modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java

```diff
@@ -223,6 +225,8 @@ public RootProjectConfigurator(Settings settings) {
 
 	@Override
 	public void apply(Project project) {
+		_applyPlugins(project);
```

Comment:
Instead of duplicating this method, let's apply the plugin [here in the WorkspacePlugin](https://github.com/liferay/liferay-portal/blob/20484e8691ac74879d65dc012c6c00ceb0390b13/modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspacePlugin.java#L81). It should have the same effect.

Reply:
OK.Got it.